es/documentation/installation/index.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: page
+title: "Instalar Ruby"
+lang: es
+---
+
+Se pueden utilizar varias herramientas para instalar Ruby.
+Esta página describe como usar los sistemas de gestión de paquetes y herramientas
+de terceros para instara Ruby y como compilarlo del código fuente.
+
```

Comment:
The attribute list must follow the paragraph without a blank line; fixed with 0d6fa617eb0c7b72f2e6c28fd6071feede203f7a .